modules/util/slice.go

```diff
@@ -34,6 +34,29 @@ func SliceContainsString(slice []string, target string, insensitive ...bool) boo
 	return SliceContains(slice, target)
 }
 
+// SliceFindString returns the index of the first occurrence of target in slice.
+// If target is not present in slice, -1 is returned.
+func SliceFindString(target string, slice []string, insensitive ...bool) int {
```

Comment:
```suggestion
func SliceFindString(slice []string, target string, insensitive ...bool) int {
```

Reply:
Actually, I don't think we need this if we have `var allAccessTokenScopeBits = map[string]AccessTokenScopeBitmap`